Module not found: Can't resolve 'React' in '/srv/node_modules/semantic-ui-react/dist/umd'

## Bug Report

### Steps
Upgrade to 0.87 from 0.86

1. `yarn upgrade semantic-ui-react@0.87`
2. restart webpack

### Expected Result
webpack dev server compiles

### Actual Result
```
Failed to compile
./node_modules/semantic-ui-react/dist/umd/semantic-ui-react.min.js
Module not found: Can't resolve 'React' in '/srv/node_modules/semantic-ui-react/dist/umd'
This error occurred during the build time and cannot be dismissed.
```

### Version
0.87.0

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "main": "dist/commonjs/index.js",
   "module": "dist/es/index.js",
   "types": "index.d.ts",
+  "unpkg": "dist/umd/semantic-ui-react.min.js",
   "files": [
     "src",
     "dist",

--- a/package.json
+++ b/package.json
@@ -5,7 +5,6 @@
   "jsnext:main": "dist/es/index.js",
   "main": "dist/commonjs/index.js",
   "module": "dist/es/index.js",
-  "browser": "dist/umd/semantic-ui-react.min.js",
   "types": "index.d.ts",
   "files": [
     "src",

